locales/en/public.json

```diff
@@ -108,31 +108,31 @@
       }
     },
     "JFR_METRICS_CARD_DESCRIPTION": "Display common performance metrics from recent JFR data.",
-    "JFR_METRICS_CARD_DESCRIPTION_FULL": "Display a single performance metric from a list of supported JFR event types. Data is displayed from the present moment back to a specified duration. Note: these metrics are only available for a single target at a time. Second Cryostat-Web instances, for the same user or others, will not behave as expected when using this card type.",
+    "JFR_METRICS_CARD_DESCRIPTION_FULL": "Display a single performance metric from a list of supported JFR Event Types. Data is displayed from the present moment back to a specified duration. Note: these metrics are only available for a single target at a time. Second Cryostat-Web instances, for the same user or others, will not behave as expected when using this card type.",
```

Comment:
I think the original `event types` makes sense here. This isn't referring to a specific term or a kind of custom resource, just the general notion of types of events that exist within JFR.